File: C/readability/readability.c
#include <ctype.h>
#include <cs50.h>
#include <stdio.h>
#include <string.h>
#include <math.h>

int main()
{
    string text = get_string("Text:");

    int letters = 0;
    int word = 1;
    int sentece = 0;

    for (int i = 0; i < strlen(text);  i++)

    {
        if (isalpha(text[i]))
        {
            letters++;
        }
        else if (text[i] == ' ')
        {
            word++;
        }
        else if (text[i] == '?' || text[i] == '!' || text[i] == '.')
        {
            sentece++;
        }
    }
    float L = letters / (float)word * 100;
    float S = sentece / (float)word * 100;

    int index = round(0.0588 * L - 0.296 * S - 15.8);
    if (index < 1)
    {
        printf("Before Grade 1");
    }
    else if (index > 16)
    {
        printf("Grade 16+");
    }
    else
    {
        printf(" Grade %i", index);
    }
    printf("\n");

}

File: python/sentimental-readability/readability.py
from cs50 import get_string
# ask user for text
text = get_string("Text : ").strip()

letters, word, sentece = 0, 1, 0
# loop trough text
for i in range(len(text)):

    if text[i].isalpha():
        letters += 1

    elif text[i] == " ":
        word += 1

    elif text[i] == "?" or text[i] == "!" or text[i] == ".":
        sentece += 1
# Coleman - Liau formula
L = float(letters / word) * 100
S = float(sentece / word) * 100
index = round(0.0588 * L - 0.296 * S - 15.8)
# print grade
if index < 1:
    print("Before Grade 1")
elif index > 16:
    print("Grade 16+")
else:
    print(f"Grade :{index}")

